test/common/storagelib.py

```diff
@@ -438,10 +438,10 @@ def udisks_objects(self):
                 "GetManagedObjects", "", [])))""")]))
 
     def configuration_field(self, dev, tab, field):
-        all = self.udisks_objects()
-        for path in all:
-            if "org.freedesktop.UDisks2.Block" in all[path]:
-                iface = all[path]["org.freedesktop.UDisks2.Block"]
+        managerObjects = self.udisks_objects()
+        for path in managerObjects:
+            if "org.freedesktop.UDisks2.Block" in managerObjects[path]:
```

Comment:
This is rather bad style anyway -- should be `for path, obj in self.udisks_objects().items()`, or possibly even `for obj in self.udisks_objects.values()` if `path` isn't used anywhere else.